packages/runtime/src/ctx.ts

```diff
@@ -142,11 +144,12 @@ export class Ctx implements Context {
   addAsset (name: string, fromAccountAddr: AccountAddress, flags: ASADeploymentFlags): number {
     const senderAcc = this.getAccount(fromAccountAddr);
 
-    // create asset
+    // create asset(with holding) in sender account
     const asset = senderAcc.addAsset(
       ++this.state.assetCounter, name,
       this.runtime.loadedAssetsDefs[name]
     );
+    this.assertMinBalance(0n, fromAccountAddr);
```

Comment:
this is not quite clear, i think it is verifying if current balance is greater than minBalance, Please update the function to be more clear

Reply:
ok, let's create another function for this